### ISSUE TITLE
App connects to server only once after fresh install

### ISSUE THREAD

**Issue by JPsychotic** [jellyfin/jellyfin-android#1392]
### Describe the bug

I installed the jellyfin app from the playstore. After the install I can connect to my server and login is successfull. After exiting the app and restarting it it can't connect to my server anymore. Only clearing cache and deleting local data allows me to then connect to the server again but only for one app lifecycle.

I attached some logs from my android device for a failed restart of the app. I noticed that some service worker could not be started but this same error is logged for the first successfull connection.

Access to the web UI via browser on the phone is possible and I stay logged in. Findroid can also connect to my server and stay logged in.

### Logs

```shell
05-21 09:39:24.162 18918 18918 I ConnectionHelper: checkServerUrlAndConnection https://jellyfin.example.de
05-21 09:39:24.163 18918 18918 I ConnectionHelper: Address candidates are [https://jellyfin.example.de, https://jellyfin.example.de:8096, https://jellyfin.example.de:8920]
05-21 09:39:24.172 18918 18918 D CompatibilityChangeReporter: Compat change id reported: 258825825; UID 10476; state: ENABLED
05-21 09:39:24.186 18918 19100 D TrafficStats: tagSocket(235) with statsTag=0xffffffff, statsUid=-1
05-21 09:39:24.189 18918 19098 D TrafficStats: tagSocket(236) with statsTag=0xffffffff, statsUid=-1
05-21 09:39:24.190 18918 19099 D TrafficStats: tagSocket(235) with statsTag=0xffffffff, statsUid=-1
05-21 09:39:24.234 18918 18918 I ConnectionHelper: Found valid server at https://jellyfin.example.de with rating GREAT and version 10.9.2
05-21 09:39:24.248 18918 18918 I WebViewUtilsKt: WebView user agent is Mozilla/5.0 (Linux; Android 14; Pixel 7 Build/AP1A.240505.005; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/124.0.6367.179 Mobile Safari/537.36, detected version is 124
05-21 09:39:24.278 18984 19018 I chromium: [INFO:viewport_data.cc(133)] E2E_Used ViewportFitCover
05-21 09:39:24.388 18918 18918 I WebView : de-DE, https://jellyfin.example.de/web/main.jellyfin.bundle.js?3b140e0ac043bcbb303b?deferred=true&ts=1716277164324 (2)
05-21 09:39:24.410 18918 18918 I WebView : Stored JSON credentials: {"Servers":[{"DateLastAccessed":1716277163089,"LastConnectionMode":2,"ManualAddress":"https://jellyfin.example.de","Name":"Eriador","Id":"8cdc266f13c948b393d0d33b969980a1","LocalAddress":"http://127.0.0.1:8096","AccessToken":"ae44d617c2c24345a62b076fdb3d8e6f","UserId":"133f7510d57a406ea572df59e085e201"}]}, https://jellyfin.example.de/web/node_modules.jellyfin-apiclient.bundle.js?3b140e0ac043bcbb303b (2)
05-21 09:39:24.410 18918 18918 I WebView : Begin ConnectionManager constructor, https://jellyfin.example.de/web/node_modules.jellyfin-apiclient.bundle.js?3b140e0ac043bcbb303b (2)
05-21 09:39:24.413 18918 18918 I WebView : returning instance from getOrAddApiClient, https://jellyfin.example.de/web/node_modules.jellyfin-apiclient.bundle.js?3b140e0ac043bcbb303b (2)
05-21 09:39:24.415 18918 18918 I WebView : jellyfin-web version 10.9.2 build 10.9.2, https://jellyfin.example.de/web/main.jellyfin.bundle.js?3b140e0ac043bcbb303b?deferred=true&ts=1716277164324 (2)
05-21 09:39:24.422  1361  1361 D usf_sensor_hal: game rotation vector: Enter Batch. period = 16666000, latency = 0.
05-21 09:39:24.422  1361  1361 D usf_sensor_hal: game rotation vector: Enter Activate. Enable = 1.
05-21 09:39:24.423  1545  1545 D AOC     : A3:MSG: usf_sensor.cc, 1677: USF: Received start sampling request for Game Rotation Vector Sensor (active:1).
05-21 09:39:24.423  1545  1545 D AOC     : A3:MSG: vsc_sensor.cc, 37: VSC: [Game Rotation Vector Sensor] enable: period 16666000, latency = 0
05-21 09:39:24.423  1545  1545 D AOC     : A3:MSG: usf_vsc_log.cc, 44: USF: {Fusion} configure fusion 15 16666us 0us.
05-21 09:39:24.424  1545  1545 D AOC     : A3:MSG: vsc_controller.cc, 532: VSC: [Game Rotation Vector Sensor] requestSensorEvents: sensor 0x10001, period=16 late
05-21 09:39:24.424  1545  1545 D AOC     : A3:MSG: usf_sensor.cc, 1677: USF: Received start sampling request for LSM6DSV Accelerometer (active:4).
05-21 09:39:24.425  1545  1545 D AOC     : A3:MSG: lsm6dsv_device.cc, 741: USF: Lsm6dsvDevice: Sensor 0: Enabled 1, odr_index 6, period_ns 16666000.
05-21 09:39:24.425  1545  1545 D AOC     : A3:MSG: lsm6dsv_fifo.cc, 755: USF: Lsm6dsvFifo: ODR: XL/G/T/TS: 120/60/15/15 WM: 1 Intr: 120.
05-21 09:39:24.426  1545  1545 D AOC     : A3:MSG: lsm6dsv_device.cc, 650: USF: Lsm6dsvDevice: Fifo was disabled for 507772ns.
05-21 09:39:24.427  1545  1545 D AOC     : A3:MSG: vsc_controller.cc, 532: VSC: [Game Rotation Vector Sensor] requestSensorEvents: sensor 0x10004, period=5 laten
05-21 09:39:24.429  1545  1545 D AOC     : A3:MSG: usf_sensor.cc, 1677: USF: Received start sampling request for LSM6DSV Gyroscope (active:2).
05-21 09:39:24.429  1545  1545 D AOC     : A3:MSG: lsm6dsv_device.cc, 741: USF: Lsm6dsvDevice: Sensor 1: Enabled 1, odr_index 7, period_ns 5000000.
05-21 09:39:24.429  1545  1545 D AOC     : A3:MSG: lsm6dsv_fifo.cc, 755: USF: Lsm6dsvFifo: ODR: XL/G/T/TS: 120/240/15/30 WM: 1 Intr: 240.
05-21 09:39:24.430  1545  1545 D AOC     : A3:MSG: lsm6dsv_device.cc, 650: USF: Lsm6dsvDevice: Fifo was disabled for 552816ns.
05-21 09:39:24.433  1361  1361 D usf_sensor_hal: game rotation vector: Enter Activate. Enable = 0.
05-21 09:39:24.433  1545  1545 D AOC     : A3:MSG: usf_sensor.cc, 1685: USF: Received stop sampling request for Game Rotation Vector Sensor (active:0).
05-21 09:39:24.434  1545  1545 D AOC     : A3:MSG: vsc_sensor.cc, 68: VSC: [Game Rotation Vector Sensor] disable.
05-21 09:39:24.434  1545  1545 D AOC     : A3:MSG: vsc_controller.cc, 550: VSC: [Game Rotation Vector Sensor] stopSensorEvents: sensor 0x10001
05-21 09:39:24.434  1545  1545 D AOC     : A3:MSG: usf_sensor.cc, 1685: USF: Received stop sampling request for LSM6DSV Accelerometer (active:3).
05-21 09:39:24.435  1545  1545 D AOC     : A3:MSG: lsm6dsv_device.cc, 741: USF: Lsm6dsvDevice: Sensor 0: Enabled 1, odr_index 5, period_ns 20000000.
05-21 09:39:24.435  1545  1545 D AOC     : A3:MSG: lsm6dsv_fifo.cc, 755: USF: Lsm6dsvFifo: ODR: XL/G/T/TS: 60/240/15/30 WM: 1 Intr: 240.
05-21 09:39:24.435  1545  1545 D AOC     : A3:MSG: lsm6dsv_device.cc, 252: USF: Lsm6dsvDevice: Config sampling too slow. Took 582642ns.
05-21 09:39:24.436  1545  1545 D AOC     : A3:MSG: lsm6dsv_device.cc, 650: USF: Lsm6dsvDevice: Fifo was disabled for 582642ns.
05-21 09:39:24.437  1545  1545 D AOC     : A3:MSG: vsc_controller.cc, 550: VSC: [Game Rotation Vector Sensor] stopSensorEvents: sensor 0x10004
05-21 09:39:24.437  1545  1545 D AOC     : A3:MSG: usf_sensor.cc, 1685: USF: Received stop sampling request for LSM6DSV Gyroscope (active:1).
05-21 09:39:24.438  1545  1545 D AOC     : A3:MSG: lsm6dsv_device.cc, 741: USF: Lsm6dsvDevice: Sensor 1: Enabled 1, odr_index 5, period_ns 20000000.
05-21 09:39:24.438  1545  1545 D AOC     : A3:MSG: lsm6dsv_fifo.cc, 755: USF: Lsm6dsvFifo: ODR: XL/G/T/TS: 60/60/15/7 WM: 6 Intr: 24.
05-21 09:39:24.439  1545  1545 D AOC     : A3:MSG: lsm6dsv_device.cc, 650: USF: Lsm6dsvDevice: Fifo was disabled for 510905ns.
05-21 09:39:24.441 18918 18918 I WebView : loading installed plugins, https://jellyfin.example.de/web/main.jellyfin.bundle.js?3b140e0ac043bcbb303b?deferred=true&ts=1716277164324 (2)
05-21 09:39:24.441 18918 18918 I WebView : [object Object], https://jellyfin.example.de/web/main.jellyfin.bundle.js?3b140e0ac043bcbb303b?deferred=true&ts=1716277164324 (2)
05-21 09:39:24.452 18918 18918 I WebView : Loading plugin (via window): NavigationPlugin, https://jellyfin.example.de/web/main.jellyfin.bundle.js?3b140e0ac043bcbb303b?deferred=true&ts=1716277164324 (2)
05-21 09:39:24.452 18918 18918 I WebView : Loading plugin (via window): ExoPlayerPlugin, https://jellyfin.example.de/web/main.jellyfin.bundle.js?3b140e0ac043bcbb303b?deferred=true&ts=1716277164324 (2)
05-21 09:39:24.452 18918 18918 I WebView : Loading plugin (via window): ExternalPlayerPlugin, https://jellyfin.example.de/web/main.jellyfin.bundle.js?3b140e0ac043bcbb303b?deferred=true&ts=1716277164324 (2)
05-21 09:39:24.453 18918 18918 I WebView : returning instance from getOrAddApiClient, https://jellyfin.example.de/web/node_modules.jellyfin-apiclient.bundle.js?3b140e0ac043bcbb303b (2)
05-21 09:39:24.453 18918 18918 I WebView : Begin connect, https://jellyfin.example.de/web/node_modules.jellyfin-apiclient.bundle.js?3b140e0ac043bcbb303b (2)
05-21 09:39:24.453 18918 18918 I WebView : Begin getAvailableServers, https://jellyfin.example.de/web/node_modules.jellyfin-apiclient.bundle.js?3b140e0ac043bcbb303b (2)
05-21 09:39:24.457 18918 18918 I WebView : Begin connectToServers, with 1 servers, https://jellyfin.example.de/web/node_modules.jellyfin-apiclient.bundle.js?3b140e0ac043bcbb303b (2)
05-21 09:39:24.457 18918 18918 I WebView : begin connectToServer, https://jellyfin.example.de/web/node_modules.jellyfin-apiclient.bundle.js?3b140e0ac043bcbb303b (2)
05-21 09:39:24.458 18918 18918 I WebView : tryReconnect: http://127.0.0.1:8096|https://jellyfin.example.de, https://jellyfin.example.de/web/node_modules.jellyfin-apiclient.bundle.js?3b140e0ac043bcbb303b (2)
05-21 09:39:24.460 18918 18918 I WebView : getTryConnectPromise http://127.0.0.1:8096, https://jellyfin.example.de/web/node_modules.jellyfin-apiclient.bundle.js?3b140e0ac043bcbb303b (2)
05-21 09:39:24.460 18918 18918 I WebView : ConnectionManager requesting url: http://127.0.0.1:8096/system/info/public, https://jellyfin.example.de/web/node_modules.jellyfin-apiclient.bundle.js?3b140e0ac043bcbb303b (2)
05-21 09:39:24.460 18918 18918 I WebView : fetchWithTimeout: timeoutMs: 20000, url: http://127.0.0.1:8096/system/info/public, https://jellyfin.example.de/web/node_modules.jellyfin-apiclient.bundle.js?3b140e0ac043bcbb303b (2)
05-21 09:39:24.486 18918 18918 E JellyfinWebViewClient: Received WebView error -6 at http://127.0.0.1:8096/system/info/public: net::ERR_CONNECTION_REFUSED
05-21 09:39:24.498 18918 18918 I WebView : fetchWithTimeout: timed out connecting to url: http://127.0.0.1:8096/system/info/public, https://jellyfin.example.de/web/node_modules.jellyfin-apiclient.bundle.js?3b140e0ac043bcbb303b (2)
05-21 09:39:24.498 18918 18918 I WebView : ConnectionManager request failed to url: http://127.0.0.1:8096/system/info/public, https://jellyfin.example.de/web/node_modules.jellyfin-apiclient.bundle.js?3b140e0ac043bcbb303b (2)
05-21 09:39:24.498 18918 18918 I WebView : Reconnect failed to http://127.0.0.1:8096, https://jellyfin.example.de/web/node_modules.jellyfin-apiclient.bundle.js?3b140e0ac043bcbb303b (2)
05-21 09:39:24.514 18918 18964 D TrafficStats: tagSocket(149) with statsTag=0xffffffff, statsUid=-1
05-21 09:39:24.527 18918 18918 I WebView : loading installed plugins, https://jellyfin.example.de/web/main.jellyfin.bundle.js?3b140e0ac043bcbb303b?deferred=true&ts=1716277164324 (2)
05-21 09:39:24.527 18918 18918 I WebView : loading packages, https://jellyfin.example.de/web/main.jellyfin.bundle.js?3b140e0ac043bcbb303b?deferred=true&ts=1716277164324 (2)
05-21 09:39:24.527 18918 18918 I WebView : loading packages, https://jellyfin.example.de/web/main.jellyfin.bundle.js?3b140e0ac043bcbb303b?deferred=true&ts=1716277164324 (2)
05-21 09:39:24.555 18918 18918 I WebView : Applied custom cast interface to window, https://www.gstatic.com/cv/js/sender/v1/cast_sender.js (1381)
05-21 09:39:24.670  7049 19016 I TcpProberController: [gms_cast_prober-1] Failed to connect to socket address: /192.168.178.26, port: 8009
05-21 09:39:24.674  7049 19016 I CastSocketMultiplexer: Stopping multiplexer...
05-21 09:39:25.031 18918 18918 I WebView : getTryConnectPromise https://jellyfin.example.de, https://jellyfin.example.de/web/node_modules.jellyfin-apiclient.bundle.js?3b140e0ac043bcbb303b (2)
05-21 09:39:25.032 18918 18918 I WebView : ConnectionManager requesting url: https://jellyfin.example.de/system/info/public, https://jellyfin.example.de/web/node_modules.jellyfin-apiclient.bundle.js?3b140e0ac043bcbb303b (2)
05-21 09:39:25.032 18918 18918 I WebView : fetchWithTimeout: timeoutMs: 20000, url: https://jellyfin.example.de/system/info/public, https://jellyfin.example.de/web/node_modules.jellyfin-apiclient.bundle.js?3b140e0ac043bcbb303b (2)
05-21 09:39:25.068 18918 18918 I WebView : fetchWithTimeout: succeeded connecting to url: https://jellyfin.example.de/system/info/public, https://jellyfin.example.de/web/node_modules.jellyfin-apiclient.bundle.js?3b140e0ac043bcbb303b (2)
05-21 09:39:25.068 18918 18918 I WebView : ConnectionManager response status: 200, url: https://jellyfin.example.de/system/info/public, https://jellyfin.example.de/web/node_modules.jellyfin-apiclient.bundle.js?3b140e0ac043bcbb303b (2)
05-21 09:39:25.069 18918 18918 I WebView : Reconnect succeeded to https://jellyfin.example.de, https://jellyfin.example.de/web/node_modules.jellyfin-apiclient.bundle.js?3b140e0ac043bcbb303b (2)
05-21 09:39:25.069 18918 18918 I WebView : ConnectionManager requesting url: https://jellyfin.example.de/System/Info, https://jellyfin.example.de/web/node_modules.jellyfin-apiclient.bundle.js?3b140e0ac043bcbb303b (2)
05-21 09:39:25.092 18918 18918 I WebView : ConnectionManager response status: 200, url: https://jellyfin.example.de/System/Info, https://jellyfin.example.de/web/node_modules.jellyfin-apiclient.bundle.js?3b140e0ac043bcbb303b (2)
05-21 09:39:25.096 18918 18918 I WebView : returning instance from getOrAddApiClient, https://jellyfin.example.de/web/node_modules.jellyfin-apiclient.bundle.js?3b140e0ac043bcbb303b (2)
05-21 09:39:25.097 18918 18918 I WebView : Setting server address to https://jellyfin.example.de, https://jellyfin.example.de/web/node_modules.jellyfin-apiclient.bundle.js?3b140e0ac043bcbb303b (2)
05-21 09:39:25.097 18918 18918 I WebView : Requesting url without automatic networking: https://jellyfin.example.de/Sessions/Capabilities/Full, https://jellyfin.example.de/web/node_modules.jellyfin-apiclient.bundle.js?3b140e0ac043bcbb303b (2)
05-21 09:39:25.099 18918 18918 I WebView : calling apiClient.ensureWebSocket, https://jellyfin.example.de/web/node_modules.jellyfin-apiclient.bundle.js?3b140e0ac043bcbb303b (2)
05-21 09:39:25.099 18918 18918 I WebView : opening web socket with url: wss://jellyfin.example.de/socket?api_key=xxx&deviceId=3fd2b42dd7905ea4, https://jellyfin.example.de/web/node_modules.jellyfin-apiclient.bundle.js?3b140e0ac043bcbb303b (2)
05-21 09:39:25.100 18918 18918 I WebView : Requesting https://jellyfin.example.de/Users/133f7510d57a406ea572df59e085e201, https://jellyfin.example.de/web/node_modules.jellyfin-apiclient.bundle.js?3b140e0ac043bcbb303b (2)
05-21 09:39:25.101 18918 18918 I WebView : returning instance from getOrAddApiClient, https://jellyfin.example.de/web/node_modules.jellyfin-apiclient.bundle.js?3b140e0ac043bcbb303b (2)
05-21 09:39:25.101 18918 18918 I WebView : Requesting https://jellyfin.example.de/DisplayPreferences/usersettings?userId=133f7510d57a406ea572df59e085e201&client=emby, https://jellyfin.example.de/web/node_modules.jellyfin-apiclient.bundle.js?3b140e0ac043bcbb303b (2)
05-21 09:39:25.137 18918 18918 I WebView : de-DE, https://jellyfin.example.de/web/main.jellyfin.bundle.js?3b140e0ac043bcbb303b?deferred=true&ts=1716277164324 (2)
05-21 09:39:25.138 18918 18918 I WebView : resolving connectToServers with result.State: SignedIn, https://jellyfin.example.de/web/node_modules.jellyfin-apiclient.bundle.js?3b140e0ac043bcbb303b (2)
05-21 09:39:25.160 18918 18918 I WebView : web socket connection opened, https://jellyfin.example.de/web/node_modules.jellyfin-apiclient.bundle.js?3b140e0ac043bcbb303b (2)
05-21 09:39:25.161 18918 18918 I WebView : Requesting https://jellyfin.example.de/Sessions?ControllableByUserId=133f7510d57a406ea572df59e085e201, https://jellyfin.example.de/web/node_modules.jellyfin-apiclient.bundle.js?3b140e0ac043bcbb303b (2)
05-21 09:39:25.164 18918 18918 I WebView : Sending web socket message: KeepAlive, https://jellyfin.example.de/web/node_modules.jellyfin-apiclient.bundle.js?3b140e0ac043bcbb303b (2)
05-21 09:39:25.167 18918 18918 I WebView : error registering serviceWorker: TypeError: Failed to register a ServiceWorker for scope ('https://jellyfin.example.de/web/') with script ('https://jellyfin.example.de/web/serviceworker.js'): A bad HTTP response code (404) was received when fetching the script., https://jellyfin.example.de/web/main.jellyfin.bundle.js?3b140e0ac043bcbb303b?deferred=true&ts=1716277164324 (2)
```

### Application version

2.6.1

### Where did you install the app from?

Google Play

### Device information

Google Pixel 7

### Android version

Android 14

### Jellyfin server version

10.9.2

### Which video player implementations does this bug apply to?

- [ ] Web player (default)
- [ ] Integrated player (ExoPlayer)
- [ ] External player (VLC, mpv, MX Player)

**Comment by nielsvanvelzen** [comment]
Closing as duplicate of #1382. This is caused by an issue in jellyfin-web that is fixed in 10.9.3.
